src/plugins/guided_onboarding/public/components/guide_button.tsx

```diff
@@ -58,7 +58,7 @@ export const GuideButton = ({
     <EuiButton
       isLoading={isLoading}
       onClick={toggleGuidePanel}
-      color="success"
+      color="accent"
```

Comment:
❓ Is this expected? Or should it rather be `accentSecondary`? @andreadelrio 
I thought `accent` buttons shouldn't really be used? 🤔

Reply:
My understanding is that we don't know for certain if we will make `accentSecondary` an available button given the numerous concerns raised by designers about having two greens (particularly in buttons). Therefore I think it's safer to use the `accent` button which has existed in the system for a long time.

Reply:
Ok, yeah I think I was behind on latest button color decisions 😅 I also just saw this ([message](https://elastic.slack.com/archives/C073NM7NGLD/p1733939238479429)).